lib/sinon/util/core/iterable-to-string.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+var typeOf = require("../../typeOf");
+
+module.exports = function (obj) {
```

Comment:
To aid in debugging, especially in less capable JavaScript engines, it is a good practice to name essential functions. I would suggest naming this one `iterableToString`.